lib/CMakeLists.txt

```diff
@@ -63,6 +65,77 @@ llvm_add_library(mull
   LLVMX86Info
 )
 
+set(mull_link_flags "") # to be filled later
+set(mull_rpath "")      # to be filled later
+
+if(MULL_SUPPORT_RUST)
+  message(STATUS "Mull> MULL_SUPPORT_RUST detected.")
+
+  set(mull_sources
+    ${mull_sources}
+    Rust/RustTest.cpp
+    Rust/RustTestFinder.cpp
+    Rust/RustTestRunner.cpp)
+
+  set(mull_header_dirs
+    ${mull_header_dirs}
+    ${MULL_INCLUDE_DIR}/Rust)
+
+  set(rust_cmake_path "${CMAKE_CURRENT_SOURCE_DIR}/../.mull.rust.cmake")
+  IF(NOT EXISTS ${rust_cmake_path})
+    message(FATAL_ERROR "MULL_SUPPORT_RUST is enabled but .mull.rust.cmake \
+      file was not found at \
+      path: \${CMAKE_CURRENT_SOURCE_DIR}/../.Mull.Rust.cmake.")
```

Comment:
Could you add one more line, e.g. "Please, take .mull.rust.cmake.example and modify according to your system".
Also, lowercase.